src/main/java/com/zendesk/maxwell/row/RowMap.java

```diff
@@ -254,6 +261,11 @@ public String toJSON(MaxwellOutputConfig outputConfig) throws Exception {
 
 		g.writeStringField("database", this.database);
 		g.writeStringField("table", this.table);
+
+		if ( outputConfig.includesRowQuery ) {
+			g.writeStringField("query", this.rowQuery);
```

Comment:
I'm not sure about your use case, but barring a good reason I'd rather see `rowQuery` be `null` instead of an empty string when we don't have one, and this property just wouldn't be set in that case. Emitting `query: ""` seems a bit unnecessary.

Reply:
Right, my mistake.

Reply:
You're still passing around `""` as the default rowQuery in a few places, this would need to be `null` instead to have it not appear in the output.

Reply:
Fixed.